src/migrations/20240221082758-action-events.js

```diff
@@ -0,0 +1,33 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS action_set_events
+        (
+            id SERIAL PRIMARY KEY NOT NULL,
+            action_set_id INTEGER NOT NULL,
+            observable_event_id INTEGER NOT NULL,
+            created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+            state TEXT NOT NULL,
+            observable_event JSONB NOT NULL,
+            action_set JSONB NOT NULL
+        );
+
+        CREATE INDEX IF NOT EXISTS idx_action_set_events_action_set_id ON action_set_events(action_set_id);
```

Comment:
This first index is redundant as Postgres can use the compound index (third one): `idx_action_set_events_action_set_id_state`.

Reply:
Sure, addressed in https://github.com/Unleash/unleash/pull/6298/commits/5e63801a1da4dd28d08db19e653aeea2f6254e12